mitosheet/mitosheet/tests/step_performers/test_export_to_file.py

```diff
@@ -490,7 +523,7 @@ def test_transpiled_with_export_to_xlsx_format_rows_no_header():
         '#b9abff',
         'B2',
         'C2',
-        True
+        False
```

Comment:
I think this might be breaking the CI - for some reason when I run this test locally, the numpy import isn't added, but on the CI it is?